src/CosmosDBStudio.ViewModel/TreeNodes/AccountFolderNodeViewModel.cs

```diff
@@ -59,11 +60,9 @@ protected override Task<IEnumerable<TreeNodeViewModel>> LoadChildrenAsync()
                     CosmosAccountFolder folder => (TreeNodeViewModel)_viewModelFactory.CreateAccountFolderNode(folder, this),
                     _ => throw new Exception("Invalid node type")
                 };
-
                 result.Add(childVM);
             }
-
-            return Task.FromResult<IEnumerable<TreeNodeViewModel>>(result);
+            return Task.FromResult<IEnumerable<TreeNodeViewModel>>(result.OrderBy(c => c.Text));
```

Comment:
I don't think this is necessary: LoadChildrenAsync is called by InternalLoadChildrenAsync, which already takes care of the ordering